interface/patient_file/summary/demographics.php

```diff
@@ -447,58 +447,11 @@ function toggleIndicator(target, div) {
         // called from stats.php.
         //
         function editScripts(url) {
-            var AddScript = function () {
-                var __this = $(this);
-                __this.find("#clearButton").css("display", "");
-                __this.find("#backButton").css("display", "");
-                __this.find("#addButton").css("display", "none");
-
-                var iam = top.frames.editScripts;
-                iam.location.href = '<?php echo $GLOBALS['webroot'] ?>/controller.php?prescription&edit&id=0&pid=' + <?php echo js_url($pid); ?>;
-            };
-            var ListScripts = function () {
-                var __this = $(this);
-                __this.find("#clearButton").css("display", "none");
-                __this.find("#backButton").css("display", "none");
-                __this.find("#addButton").css("display", "");
-                var iam = top.frames.editScripts
-                iam.location.href = '<?php echo $GLOBALS['webroot'] ?>/controller.php?prescription&list&id=' + <?php echo js_url($pid); ?>;
-            };
 
             let title = <?php echo xlj('Prescriptions'); ?>;
             let w = 960; // for weno width
 
             dlgopen(url, 'editScripts', w, 400, '', '', {
-                buttons: [{
-                    text: <?php echo xlj('Add'); ?>,
```

Comment:
can use promised then here

Reply:
Sounds good, made the change, since refreshme is used in multiple places, I converted to promise and called the function.